chapters/102/index.md

```diff
@@ -0,0 +1,124 @@
+
+![](images/query_flow.svg)
+
+
+### Inside a Flow
+
+Here, we’ll use the indexing Flow as an example:
+
+![](images/flow_pods.svg)
+
+In this Flow, different Pods perform different tasks:
+
+1. Input Documents segmented into chunks (sentences)
+2. Chunks transformed into vector embeddings
+3. Chunks indexed in key-value pairs and saved to storage
+
+### Inside a Pod
+
+![](images/pod_peas.svg)
+
+In the Segmenter Pod, there’s only one Pea. But sometimes you group multiple copies of the same Peas in a Pod to increase reliability or [performance](link to sharding section).
+
+### Inside a Pea
+
+Let’s take a closer look at the Pea in the Segmenter Pod:
+
+![](images/pea_exec_driver.svg)
+
+As you can see, a Pea is a wrapper for an Executor and its Driver. 
+
+While Peas and Pods perform the communication in a Flow, the messages themselves come from the Executors. And each Executor needs a specific Driver to handle its input and output messages.
+
+You can see an example of how Flow, Pods, Peas, Executors, and Drivers work together in our [Wikipedia sentence search example](https://github.com/jina-ai/examples/tree/master/wikipedia-sentences).
+
+## YAML Configuration
+
+From Flows to Executors, every part of Jina is configurable with YAML files. YAML files let you change the behavior of an object without touching its code.
+
+![](images/yaml.svg)
+
+Besides YAML, you can also design Flows in [Jina Dashboard](http://dashboard.jina.ai) or [Python code](link to Flow IO section).
+
+
+## Search Modality
+
+Our example above shows searching using a single type of data, but what about going further?
+
+
+### Cross-modal search
+
+An example of cross-modal search is using text to search for images, or the other way around. 
+
+![](images/cross_modal.svg)
+
+
+### Multi-modal search
+
+You can use multiple modalities as input for your search applications. For example, a search query input can be an image plus some descriptive text.
+
+![](images/multi_modal.svg)
+
+To implement cross-modality or multi-modality, visit the [dedicated section](link) in our documentation.
+
+
+## JinaD
+
+[JinaD]() (Jina Daemon) enables orchestration and management of Jina Flows in distributed search systems through an API endpoint.
+
+
+## Jina Suite
+
+Jina Suite consists of several different open-source products - Core, Hub, Dashboard and Box.
+
+What you’ve learned in the previous sections are the most important components in Core (Jina Core). But there’s more to explore.
+
+
+![](images/suite.svg)
+
+
+### Core
+
+You construct and manage your search workflows in Core. It consists of distributed microservices (Flows, Peas, Pods, and Executors, etc.) Optionally, you can use JinaD to orchestrate these microservices.
+
+
+### Hub
+
+[Jina Hub]() for hosting Jina Pods and Apps via container images. You can upload and share images of Pods with other developers in the community.
```

Comment:
The link and the following 2 are missing as well.

Reply:
Yup. Until docs URLs are finalized there's not much we can do in that regard (unless the platform offers some other kind of linking?)